client/src/modules/patients/record/bh-patient-financial-activity.html

```diff
@@ -79,6 +79,33 @@ <h5 ng-show="!$ctrl.status.isInGoodStanding" class="text-danger">
             (<span am-time-ago="$ctrl.data.aggregates.until"></span> <i class="fa fa-clock-o"></i>)
           </dd>
         </dl>
+
+        <hr/>
+        <h5 class="text-success" style="margin-top:0;">
+          <i class="fa fa-medkit" aria-hidden="true"></i> <span translate>PATIENT_RECORDS.STOCK_MOVEMENTS_TO_PATIENT</span>
+        </h5>
+
+        <div class="col-sm-6 col-xs-12">
+          <table class="table table-responsive table-condensed">
+            <thead>
+              <tr>
+                <td translate>FORM.LABELS.NR</td>
+                <td translate>FORM.LABELS.REFERENCE</td>
+                <td translate>FORM.LABELS.DATE</td>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-repeat="dataMovement in $ctrl.dataMovement">
```

Comment:
We should probably limit this to the most recent ... 5 items?  Or something like that.  We don't want the grid to go on forever!

I would just implement this on the client side for the moment.